wordpress-mobile/WordPress-iOS
# Stats Insights: preserve and use Insight types to display

Ref #10374 (part 3 of 3)

This is the last part of #10374 , allowing the Insights table the flexibility to show Insight types based on an array that (in the future) will be set by the user's Manage Insights settings.

Magic included:
- Writes and loads an array of insight types to display to/from User Defaults. 
  - Currently, all the Insight types in the default order (based on the Zeplin Insights design) will be shown.
- Creates table rows depending on the insight type array.
  - Currently there is only Latest Post Summary.
  - There are stub `DDLogDebug`s for the others. More ✨ to come!

To test:
- With a fresh install, go to Stats > Insights.
  - Verify the Latest Post Summary displays and is correct.
- Sanity checks on these three states to make sure all works as expected. 
  - Post has data.
  - Post has no data.
  - Site has no posts.
